### DISCUSSION TITLE
How to install specific az version in the mixin

### DISCUSSION THREAD

**Discussion by tamirkamara** [getporter/porter#2135] [Q&A]
Hi,
Is it possible to install a specific version of the az-cli within the mixin, much like the [terraform one enables](https://github.com/getporter/terraform-mixin#clientversion) us to do?

**Comment by carolynvs** [comment]
It's not but I can get that implemented pretty quickly. It's just an oversight that the mixin doesn't support setting the client version.

**Reply by carolynvs** [reply]
I've got a start that seems to work. I'll test it this week and reply back when there's a new release.

https://github.com/getporter/az-mixin/pull/43
